include/deal.II/base/quadrature.h

```diff
@@ -221,6 +221,17 @@ class Quadrature : public Subscriptor
   template <class Archive>
   void serialize (Archive &ar, const unsigned int version);
 
+  /**
+   * Returns is_tensor_product_flag;
```

Comment:
This is not a useful description in a public function as it requires knowledge about the internal implementation of the class. I think it would be more useful if the function copied some of the description of the flag as explanation.